DQM/EcalMonitorClient/src/TrigPrimClient.cc

```diff
@@ -19,16 +19,25 @@ namespace ecaldqm {
     minEntries_ = _params.getUntrackedParameter<int>("minEntries");
     errorFractionThreshold_ = _params.getUntrackedParameter<double>("errorFractionThreshold");
     TTF4MaskingAlarmThreshold_ = _params.getUntrackedParameter<double>("TTF4MaskingAlarmThreshold");
+    sourceFromEmul_ = _params.getUntrackedParameter<bool>("sourceFromEmul");
+    if (!sourceFromEmul_) {
+      MEs_.erase(std::string("NonSingleSummary"));
+      MEs_.erase(std::string("TimingSummary"));
+      sources_.erase(std::string("EtEmulError"));
+      sources_.erase(std::string("MatchedIndex"));
+    }
   }
 
   void TrigPrimClient::producePlots(ProcessType) {
-    MESet& meTimingSummary(MEs_.at("TimingSummary"));
-    MESet& meNonSingleSummary(MEs_.at("NonSingleSummary"));
+    MESet* meNonSingleSummary = nullptr;
+    MESet* meTimingSummary = nullptr;
+    ;
```

Comment:
typo?

Reply:
You mean by the different order?
@tanmaymudholkar   ?

Reply:
I mean " ;" https://github.com/cms-sw/cmssw/blob/cd8c63c1903d02ccb7f75ee69abc41b7d4f5717f/DQM/EcalMonitorClient/src/TrigPrimClient.cc#L34

Reply:
Ohhhh, right! I missed it, sorry

Reply:
@silviodonato @jfernan2 sorry, indeed this is a typo that I missed. I've removed it in the latest commit.